ComponentKit/Core/CKCompositeComponent.mm

```diff
@@ -64,6 +67,11 @@ + (instancetype)newWithView:(const CKComponentViewConfiguration &)view size:(con
   CK_NOT_DESIGNATED_INITIALIZER();
 }
 
+- (std::vector<CKComponent *>)render:(id)state
+{
+  return {_component};
+}
+
 - (CKComponentLayout)computeLayoutThatFits:(CKSizeRange)constrainedSize
                           restrictedToSize:(const CKComponentSize &)size
                       relativeToParentSize:(CGSize)parentSize
```

Comment:
so this method is now dead code, right? why is it still here?

Reply:
What about old components that didn't migrate to the render method? the idea is to allow mix-and-match between "old" and new components.

Reply:
Should we provide a new superclass for components that want to use the render method?

Reply:
We are going to try this idea. I'm still wondering - what is the main advantage of this separation?
If we implement the render in `CKCompositeComponent` we receive part of the conversion for "free".
What is the down side of this implementation?

Reply:
Fair point @kfirapps. I think you're seeing us be really scared of interface creep. We've traditionally been really wary of components with more than a single responsibility. Perhaps we should relax some of our constraints here in favor of a new path forward. Perhaps to ease the change, what's the plan for full conversion such that we remove this? Or is it permanent?

Reply:
The equivalent component in the new API is `CKChildComponent`. That means that once we finish the migration we could get rid of this component.